policyengine_us/parameters/gov/states/ct/tax/income/subtractions/social_security/excess_fraction.yaml

```diff
@@ -0,0 +1,11 @@
+description: Connecticut calculates this percentage of the excess described in Section 86(b)(1) of the Internal Revenue Code.
```

Comment:
rephrase to name the variable that is described:
described in Section 86(b)(1) of the Internal Revenue Code.

Reply:
In the legal code, its referred to as "percentage of the social security benefits" so I added this